docs/post-install.md

```diff
@@ -0,0 +1,61 @@
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////"
+==> valkyrie:         ]
+==> valkyrie:     }
+==> valkyrie: }
+==> valkyrie: 
+==> valkyrie: TASK: [getvalkyrie.valkyrie | Output the login link] ************************** 
+==> valkyrie: ok: [localhost] => {
+==> valkyrie:     "var": {
+==> valkyrie:         "aegir_login_link_output.stdout_lines": [
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////",
+==> valkyrie:             "  You can access the front-end of your Aegir install at:",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "http://valkyrie.local/user/reset/1/1429111863/uAvC9VJx-l2CqhaufWg8ElCSFWzfdTiUTxJOQJ-pbz4/login",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////"
+==> valkyrie:         ]
+==> valkyrie:     }
+==> valkyrie: }
+```
+
+* Use the login link as a one-time access to the now deployed aegir frontend, and reset the admin password immediate.  Don't forget to tick the check boxes for all the roles you would want the admin user to be able to play on the aegir server.
+* Copy the displayed deploy ssh key, and add it to your project on github.  This will allow you to publish to the project from valkyrie.
+
+### Looking Around the system
+
+**The directory in which vagrant commands are executed is significant**
+* The default location for vagrant base boxes is ~/.vagrant/machine, but valkyrie does not use that
+* The base directory for valkyrie vagrant vm is the directory in which you invoked the command `drush vmnew valkyrie`
+* To see all vagrant instances for your user account, from anywhere type `vagrant global-status` .. the display of this command is cached and might show recently deleted VMs
+* From anywhere you can run vagrant commands against any vm, by postfixing the id displayed by vagrant global-status.  For example:
+```
+vagrant destroy vmid
+vagrant ssh vmid
+etc 
+```
+* From a given install base, you can issue vagrant commands without specifying a vm id, and the commands will be run against the vm defined in the Vagrantfile within that directory
+* You can further customize the vm, by editing the Vagrant file
+
+**Working in the VM**
```

Comment:
The purpose of Valkyrie is largely to remove the need to SSH into the VM. That said, `vagrant ssh` is, again, standard Vagrant functionality.

What is perhaps interesting is that you can access the VM via Drush, using the generated aliases. `drush sa` should list a couple special aliases for the vm itself and the Aegir site. `drush @v ssh`, for example, will SSH you into the VM as the 'aegir' user, which is usually what you'll want, to avoid messing up file permissions and such.